src/pm/xbps.rs

```diff
@@ -84,23 +84,21 @@ impl Pm for Xbps {
             .await?;
 
         let mut stdout = std::io::stdout();
-        let mut res = Ok(());
-        for line in lines {
-            match line {
-                Ok(line) => stdout.write_all(&line)?,
+        lines.into_iter().try_fold(Ok(()), |acc, line| {
+            std::io::Result::Ok(match line {
```

Comment:
I don't think you need to specify the exact result type for `Ok`

Reply:
@tranzystorek-io 
I agree with you, yet it's the Rust compiler that enforces it, not me :)